nixos/modules/programs/nist-feed.nix

```diff
@@ -0,0 +1,47 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.nist-feed;
+in
+{
+  options = {
+    programs.nist-feed = {
+      enable = mkEnableOption (mdDoc "NIST Feed notifies you about the newest published CVEs.");
+      package = mkPackageOption pkgs "nist-feed" { };
+      arguments = mkOption {
+        type = types.str;
+        default = "-l -s CRITICAL";
+        description = mdDoc ''lib.
+          Arguments to provide to NIST-Feed.
```

Comment:
Maybe you could add a link to some kind of upstream documentation as to what arguments exist? I think that would be useful for a user looking to configure the service.